o5/deployer/v1/deployment.proto

```diff
@@ -116,7 +117,11 @@ message DeploymentEventType {
     DeploymentSpec spec = 1;
   }
 
-  message Triggered {}
+  message Triggered {
+    string environment = 1;
+    CodeSourceType source = 2;
```

Comment:
It's a little bit weird, but the `TriggerDeploymentRequest` doesn't actually produce one of these events, it adds a `Created` event. So adding the source to the spec will technically cover it.

This event instead is a side effect of a Stack event. I still need to double check that we can populate this info from that event.

Reply:
At the point where the event is created, it doesn't conveniently have all the info available without fetching. Since it doesn't actually affect the details we were looking to affect, I'm going to remove this one for now and we can add more to it later as needed.